src/clasqa/QADB.groovy

```diff
@@ -519,6 +574,12 @@ class QADB {
   private double charge,chargeMin,chargeMax,chargeTotal
   private boolean chargeCounted
   private def chargeCountedFiles
+  private double chargeHL // Helicity Latched charge
```

Comment:
my mistake, this is actually the wrong type; perhaps we can just use `def`. I'm going to commit this fix to re-trigger the CI tests.
```suggestion
  private def chargeHL // Helicity Latched charge
```